client/src/firebase/FirestoreDocumentReader.js

```diff
@@ -7,8 +7,8 @@
 
```

Comment:
For future PRs can you please separate lint stuff and relevant stuff.  This PR looks a lot bigger than it is because it has a lot of unrelated lint changes.

Reply:
Yessir 👍